app/assets/javascripts/editor.js

```diff
@@ -123,25 +123,17 @@ $E = {
   generate_preview: function(id,text) {
     $('#'+id)[0].innerHTML = marked(text)
   },
-  toggle_preview: function (comment_id = null) {
+  toggle_preview: function() {
```

Comment:
I searched through `plots2`: it seems as if `toggle_preview` is never called with a `comment_id` parameter. I could be wrong about that(?) For now, I am deleting this unused code.

Reply:
Hmm, and comment previews are toggled with a different method? Ok!